### PR TITLE
Run rw without 'yarn run'

### PR DESCRIPTION
**Pull request by pepibumur**
It's a subtle thing, and perhaps subjective, but I would love to be able to run `rw` without having to prefix it with `yarn run`. To do so, I'm proposing to use, [shadowenv](https://github.com/Shopify/shadowenv), a tool that allows defining environment changes that are tied to projects. That allows exposing the `node_module/.bin` directory at the root of the project where the `rw` executable lives, and thus being able to run all the commands without `yarn run`.

This improvement also brings some safety because it prevents people from running a global Redwood version that might be different from the one the project is pinned to.

### CODE REVIEW

**Comment by Tobbe** [comment]
Obviously won't work in PowerShell or Windows Cmd, but won't hurt anything either 👍

**Comment by thedavidprice** [comment]
zOMG I've wanted this for so. long. now. https://github.com/redwoodjs/redwood/issues/328

There's already been discussion that we need to be careful in the documentation about using full command names and not abbreviations — let people discover the shortcuts once they understand the reality first. In this case, how can we make it clear what's happening under the hood? And/or would we need to? 

**tl;dr:** if we JustDoIt, is there any way we'd be unintentionally setting users up for a trap? 

From the outset, I don't see much by way of downside. Just the sweet sweet pleasure of only typing `rw`...

**Comment by Tobbe** [comment]
It'd only work for those who already have shadowenv installed, right? And if you do, I guess you know what you're doing. Only thing I guess is if you get too used to just wringin `rw ...`, and then you use another environment where shadowenv isn't available, might catch you off guard.

**Comment by jvanbaarsen** [comment]
What do you all think about making this a readme item or a recipe or something instead of putting it in the codebase? My personal opinion is that with adding these type of things you're opening yourself up to people wanting to add something similar but with tool X, or they want to have tool Y for platform Z.

**Comment by thedavidprice** [comment]
@pepibumur I'm unfamiliar with shadowenv. Could you help me understand what additional steps are needed to enable this once the file `rw.lisp` is created?  For example, it seems the user would need to:
- either run through this [Shadowenv Setup](https://shopify.github.io/shadowenv/getting-started/), or
- use something like the [VS Code Plugin](https://github.com/Shopify/vscode-shadowenv)

**Comment by pepibumur** [comment]
> @pepibumur I'm unfamiliar with shadowenv. Could you help me understand what additional steps are needed to enable this once the file rw.lisp is created? For example, it seems the user would need to:

Developers only need to have `shadowenv` installed in their system. They can install it using Homebrew. Once installed, when they cd into the directory shadowenv will tell them that there's a shadowenv declaration file and prompt them to trust it. Once they trust it, the tool loads the variables automatically in the environment.

**Comment by thedavidprice** [comment]
Got it. And thanks!

Ok, given that this is Mac-only and requires installation outside a project's node packages, here's how to proceed with this:
1. Close out this PR
2. Move forward as a Redwood Setup command, which is intended for install/instantiate/boilerplate/config/etc
- For a complex reference, see the `setup tailwind` command, [located here](https://github.com/redwoodjs/redwood/tree/main/packages/cli/src/commands/setup/tailwind). For a simple example closer to what will be required here, see the `setup webpack` command [located here](https://github.com/redwoodjs/redwood/tree/main/packages/cli/src/commands/setup/webpack).
3. This Setup command will probably only do two things 1) add `.shadowenv.d/rw.lisp` 2) console output the installation steps and link to [Shadowenv Doc](https://shopify.github.io/shadowenv/getting-started/)
4. Document the command in the `setup` section of the [CLI Doc](https://redwoodjs.com/docs/cli-commands#setup)

I do think this is valuable and worth the effort. It's just not a fit as a default for the installation template.

What do you think @pepibumur?

**Comment by thedavidprice** [comment]
Checking back on my comment above @pepibumur Any reactions and/or questions? I'm here when you’re available!

**Comment by thedavidprice** [comment]
Thanks again for suggesting this @pepibumur 

I've moved the next steps into this Issue #3094 Would be a great helper to have.
